pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -474,7 +474,8 @@ func getUnclaimedAccount(reqLogger logr.Logger, accountList *awsv1alpha1.Account
 		if !account.Status.Claimed && account.Spec.ClaimLink == "" && account.Status.State == "Ready" {
 			// Check for a reused account with matching legalEntity
 			if account.Status.Reused {
-				if matchAccountForReuse(&account, accountClaim) {
+				// #nosec
+				if matchAccountForReuse(&account, accountClaim) { // #nosec
```

Comment:
What is the linter complaining about on this line?

Reply:
It complains about implicit memory aliasing in the for loop

Reply:
Ahh, I see now.  The for loop was in the collapsed part of the code so that wasn't visible haha.  Thanks!